py-polars/polars/lazyframe/groupby.py

```diff
@@ -567,6 +567,35 @@ def n_unique(self) -> LDF:
         """
         return self.agg(F.all().n_unique())
 
+    def approx_unique(self) -> LDF:
```

Comment:
We shouldn't expose this on the `DataFrame`. I don't expect this to work with all dtypes, and we don't do this with most expressions.

Reply:
done